packages/footer/src/Footer.stories.tsx

```diff
@@ -69,66 +70,66 @@ DarkAltVariant.args = {
 };
 
 const AgComplexFooter: ComponentStory<typeof Footer> = (args) => {
-	const columnSpanning = { xs: 12, sm: 6, md: 4, lg: 3 } as const;
+	const columnSpanning = { xs: 12, sm: 6, lg: 3 } as const;
```

Comment:
Fixed a bug where content wrapped incorrectly on the `md` breakpoint.  You can see the bug here https://steelthreads.github.io/agds-next/storybook/iframe.html?id=layout-footer--complex-content&args=&viewMode=story